app/src/test/java/com/duckduckgo/app/referencetests/brokensites/BrokenSitesReferenceTest.kt

```diff
@@ -147,6 +149,7 @@ class BrokenSitesReferenceTest(private val testCase: TestCase) {
             consentSelfTestFailed = testCase.consentSelfTestFailed.toBoolean(),
             errorCodes = "",
             httpErrorCodes = "",
+            loginSite = null,
```

Comment:
These 2 changes (line 104 and line 152) need to be done in `BrokenSitesMultipleReportReferenceTest` as well.